Documentation/Signoffs/detaildesgincamera.md

```diff
@@ -0,0 +1,137 @@
+
+## Specifications
+
+1. Mounting
+    * Attachment location
+        * Light posts located around the parking lot (see Figure 2)
+        * Light posts are 25 feet tall
+    * Not allowed to drill into the posts
+        * Hose clamp for block (mounting surface) to top of light pole
+        * More refined mount will be designed by ME Team
+    * Not allowed to add additional infrastructure (i.e. posts)
+2. Monitoring
+    * Cameras
+        * All parking spots must be seen by cameras
+        * Able to recognize car from feed
+        * Have wireless communication to server
+3. Maintainability
+    * Should require minimum maintenance once installed
+4. Power
+    * All cameras will be solar powered to provide minimal maintenance
+5. Communication
+    * Wireless communication
+        * Bluetooth not an option due to range	
+        * WiFi
+            * Need consideration of following factors:
+                * Quality
+                * Range
+                * Downtimes
+
+## Analysis
+
+1. Mounting
+    * In order for the camera to operate correctly it will need to be mounted securely onto the light post. However, a constraint we received from facilities is that drilling holes in the pole is not allowed. Due to this we need a mounting method that does not require drilling and is adjustable in the event that we need to relocate the camera on the pole. For this reason we plan to use a hose clamp/vertical bracket combination similar to Dahua Technology pole mount bracket (MFR# PFA150). Currently we have not received pole diameter dimensions from facilities so the correct hose clamp size has not been determined. However, this is a minor issue that can quickly be resolved by either measuring the pole or receiving the information from facilities. Hose clamps have a wide size range that should satisfy our requirements.
+    * The cameras shall be placed no less than 5 feet below the lights on the light posts to ensure that the emanating light is not obstructed at night. Although there is very little concern that the lights will affect the cameras during the system's hours of operation, the cameras shall be placed so that they do not obstruct the light at night.
```

Comment:
This is not analysis, this is a constraint. The design can't be analyzed to verify this is the case as there is no design for this.